zoninator.php

```diff
@@ -1128,11 +1128,11 @@ function get_zones_for_post( $post_id ) {
 
 	function get_zones( $args = array() ) {
 
-		$args = wp_parse_args( $args, array(
+		$args = apply_filters( 'zoninator_terms_query_args', wp_parse_args( $args, array(
```

Comment:
I actually thought about submitting the suggestion from @bcampeau. Probably should have run that by him. At the moment I would be using this to reorder tabs/zone listing in the admin.